packages/app/src/app/pages/Pro/components/SubscriptionCard.tsx

```diff
@@ -34,7 +34,7 @@ const subLinkColors = {
   light: { base: '#0E0E0E', hover: '#161616' },
 };
 
-const StyledSubscriptionLink = styled.a<{
+export const StyledSubscriptionLink = styled.a<{
```

Comment:
are these being reused? should they be moved to a `styled.tsx` file in the folder? I would like to slowly enforce the idea of a single component being exported from a file like this (.tsx). Defining multiple styled blocks is great as long as they are local. But if they are used somewhere else I would extract them to a common file.